packages/docs/docs/docs/zeptoken.md

```diff
@@ -0,0 +1,38 @@
+---
+id: zeptoken
+title: The ZEPToken
+sidebar_label: The ZEPToken
+---
+
+ZeppelinOS' native ZEP token is used to incentivize the creation and maintenance of a healthy ecosystem of EVM packages. It does so via a series of mechanisms that are detailed in the following sections of this article.
```

Comment:
I'd mention in the very first paragraph that the ZEP token is **not** needed for using ZeppelinOS, only for participating in this ecosystem.